HomeWorks/java/lesson2/Task2HW2.java

```diff
@@ -0,0 +1,14 @@
+package lesson2;
+
+public class Task2HW2 {
+    public static void main(String[] args) {
+        String shortString = "word";
+        String longString = "helloWorld";
+        int cont1 = shortString.length();
+        int cont2 = longString.length();
+
+        System.out.println(cont1);
+        System.out.println(cont2);
```

Comment:
it's needed to concatenate these words:
 static int method(String str1, String str2) {
        int wordlenght = str1.length();
        int wordlenght1 = str2.length();
        int wordsum = wordlenght + wordlenght1;
        System.out.println(wordsum);
        return wordsum;
    }